wifi_password/utils.py

```diff
@@ -0,0 +1,206 @@
+#!/usr/bin/env python3
+import os
+import sys
+import re
+import subprocess
+import qrcode
+
+import constants
+
+def get_platform():
+    """
+    Returns the name of the platform where the application is currently running
+    """
+    platforms = {
+        'linux1': constants.LINUX,
+        'linux2': constants.LINUX,
+        'darwin': constants.MAC,
+        'win32': constants.WINDOWS
+    }
+
+    if not sys.platform in platforms:
+        return sys.platform
+
+    return platforms[sys.platform]
+
+def get_windows_profiles():
```

Comment:
Instead of having a separate function for each operating system, have a single function called `get_profiles()`, which behaves like the other functions.

Reply:
Sure I don't see any problem with it